src/Shouldly/ShouldlyMessage.cs

```diff
@@ -240,20 +240,21 @@ public string CreateExpectedErrorMessage()
         private static string CreateActualVsExpectedMessage(IShouldlyAssertionContext context)
         {
             var codePart = context.CodePart;
-            string message = string.Format(@"
-    {0}
-        {1}
-    {2}
-        but was
-    {3}",
-                codePart, context.ShouldMethod.PascalToSpaced(), context.Expected.ToStringAwesomely(), context.Actual.ToStringAwesomely());
+            var message = string.Format(@"
+            {0}
+                {1}
+            {2}
+                but was
+            {3}",
+                codePart, context.ShouldMethod.PascalToSpaced(), context.Expected.ToStringAwesomely(),
+                context.ShouldMethod == "ShouldNotBe" ? string.Empty : context.Actual.ToStringAwesomely());
```

Comment:
I wonder whether it's worth moving this into a custom message generator? Thoughts @JakeGinnivan?